test/e2e/disk_encryption.go

```diff
@@ -33,20 +32,20 @@ import (
 var _ = Describe("Encryption at host", func() {
 	It("must be enabled on the test cluster and each VM must have encryption at host enabled", func(ctx context.Context) {
 		By("getting the test cluster resource")
-		oc, err := clients.OpenshiftClustersv20220401.Get(ctx, vnetResourceGroup, clusterName)
+		oc, err := clients.OpenshiftClusters.Get(ctx, vnetResourceGroup, clusterName)
 		Expect(err).NotTo(HaveOccurred())
 
 		By("checking that encryption at host is enabled for masters")
 		Expect(oc.OpenShiftClusterProperties).To(Not(BeNil()))
 		Expect(oc.OpenShiftClusterProperties.MasterProfile).To(Not(BeNil()))
-		Expect((*oc.OpenShiftClusterProperties.MasterProfile).EncryptionAtHost).To(Equal(mgmtredhatopenshift20220401.Enabled))
+		Expect((*oc.OpenShiftClusterProperties.MasterProfile).EncryptionAtHost).To(BeEquivalentTo("Enabled"))
```

Comment:
This was actually an issue when I bumped the API version.. I got the following error:
```
Expected
    <redhatopenshift.EncryptionAtHost>: Enabled
to equal
    <redhatopenshift.EncryptionAtHost>: Enabled
```

Normally I'd agree on using a constant and not a hard coded string.. My concern though is that this is the only test that is using a versioned constant and would need to be bumped each time the API is or else it will fail (which is what `BeEquivalentTo` is meant to test). If the value of that constant changed in the future for some reason, this test would still catch that and would need to be updated. This makes the test generic where it can test multiple API versions and still ensure that the actual value that's stored in the cluster doc remains the same. Thoughts?

Reply:
I think it's fair to use a hard coded string in this case to eliminate "test toil".  If every time we bump the API being used here and forget to update the e2e test with the correct client's constant for Enabled, the test is going to fail, we're going to context switch into looking at why the test failed, realize it's because we didn't update the client, and then wait again for it to complete.

This will still catch if a change in the string is detected.  Should eliminate some churn.